### DISCUSSION TITLE
Anyone using secondary detection methods?

### DISCUSSION THREAD

**Discussion by morrowwm** [Motion-Project/motion#1775] [General]
I'm interested in trying one of the secondary_detection methods, but I don't see any existing discussion about them.

```
secondary_method
Values: haar, hog, dnn | Default: Not Defined
The secondary detection method.
```
I understand the instructions here
[https://motion-project.github.io/motionplus_examples.html#haar_model_training](https://motion-project.github.io/motionplus_examples.html#haar_model_training)
in general. But has anyone actually tried this and been successful?

**Comment by tosiara** [comment]
Probably, better to ask in https://github.com/Motion-Project/motionplus

**Reply by morrowwm** [reply]
Argh, thanks.
